stdlib/universe/filter.go

```diff
@@ -170,3 +173,29 @@ func (t *filterTransformation) UpdateProcessingTime(id execute.DatasetID, pt exe
 func (t *filterTransformation) Finish(id execute.DatasetID, err error) {
 	t.d.Finish(err)
 }
+
+// MergeFilterAnyRule removes Filter nodes whose predicate always evaluates to true.
+type MergeFilterAnyRule struct{}
```

Comment:
Do you mind renaming this rule to something like `RemoveConstantFilterRule` or `RemoveTrivialFilterRule`? I think something like that would be more descriptive.